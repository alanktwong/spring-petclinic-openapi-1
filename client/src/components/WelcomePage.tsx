import * as React from 'react';

export default () => (
  <div>
    <h2>Welcome</h2>
    <div className="row">
      <div className="col-md-12">
        <img className="img-responsive" src="/images/pets.png" alt="" />
      </div>
    </div>
  </div>
);
