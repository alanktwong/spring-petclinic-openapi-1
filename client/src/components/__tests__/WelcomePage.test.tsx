import React from 'react';
import renderer from 'react-test-renderer';
import WelcomePage from '../WelcomePage';

it('renders correctly', () => {
  const tree = renderer.create(<WelcomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
