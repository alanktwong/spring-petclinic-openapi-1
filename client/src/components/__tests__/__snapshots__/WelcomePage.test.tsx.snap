// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    Welcome
  </h2>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <img
        alt=""
        className="img-responsive"
        src="/images/pets.png"
      />
    </div>
  </div>
</div>
`;
